### PR TITLE
Add "make watch-babylon" [skip ci]

### PR DESCRIPTION
**Pull request by nicolo-ribaudo**
| Q                        | A 
| ------------------------ | ---
| Fixed Issues?            | 
| Patch: Bug Fix?          |
| Major: Breaking Change?  |
| Minor: New Feature?      |
| Tests Added + Pass?      | 
| Documentation PR         | 
| Any Dependency Changes?  |
| License                  | MIT

Please  :pray:

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -42,6 +42,10 @@ watch: clean
 	make clean-lib
 	BABEL_ENV=development ./node_modules/.bin/gulp watch
```
Comment (xtuc) [inline Makefile]: Do we want to include this in the normal `watch`?

Comment (nicolo-ribaudo) [reply Makefile]: Does make support running two commands at the same time?

Comment (xtuc) [reply Makefile]: Good point, that will probably be too complicated. LGTM then.

**Comment by xtuc** [comment]
I'll go ahead and merge this. Thanks @nicolo-ribaudo.

**Comment by hzoo** [comment]
This is totally fine, but I think we should just go with logan's suggestion in the future of compiling the individual files in watch mode and then only building out rollup/webpack for babel-standalone when publishing

**Pull request merged**